src/etl/sds/trigger/manual/tests/test_manual_trigger.py

```diff
@@ -0,0 +1,59 @@
+import json
+from collections import deque
+
+import boto3
+import pytest
+from etl_utils.constants import ETL_QUEUE_HISTORY, ETL_STATE_MACHINE_HISTORY
+from event.json import json_loads
+from mypy_boto3_lambda.type_defs import InvocationResponseTypeDef
+
+from etl.sds.worker.extract.tests.test_extract_worker import GOOD_SDS_RECORD
+from test_helpers.s3 import _set_etl_content, _set_etl_content_config
+from test_helpers.terraform import read_terraform_output
+
+TEST_DATA_NAME = "123.ldif"
+EXPECTED_CHANGELOG_NUMBER = 123
+EMPTY_LDIF_DATA = b""
+EMPTY_JSON_DATA = deque()
+
+
+@pytest.mark.timeout(20)
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    ("history_object"),
+    [
+        ("bulk.0.123.2024-07-03T14.04.12.721948"),
+        ("update.100.123.2024-07-03T14.04.12.721948"),
+    ],
+)
+def test_manual_trigger(history_object):
+    # Where the state is located
+    bucket_config = _set_etl_content_config()
+    manual_trigger_arn = read_terraform_output("manual_trigger_arn.value")
+
+    #     # Set some questions
+    s3_client = boto3.client("s3")
+    _set_etl_content(s3_client=s3_client, bucket_config=bucket_config, bulk=False)
+
+    s3_client.put_object(
+        Bucket=bucket_config["etl_bucket"],
+        Key=f"{ETL_STATE_MACHINE_HISTORY}/{history_object}",
+        Body=GOOD_SDS_RECORD,
+    )
+    s3_client.put_object(
+        Bucket=bucket_config["etl_bucket"],
+        Key=f"{ETL_QUEUE_HISTORY}/{history_object}",
```

Comment:
Is this not needed if the manual trigger only reads from the state_machine_hisotry and doesn't care about what's in the queue?